x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/action/EsqlQueryRequestTests.java

```diff
@@ -158,8 +159,13 @@ public void testRejectUnknownFields() {
             }""", "unknown field [asdf]");
     }
 
-    public void testKnownVersionIsValid() throws IOException {
+    public void testKnownStableVersionIsValid() throws IOException {
         for (EsqlVersion version : EsqlVersion.values()) {
+            if (version == EsqlVersion.SNAPSHOT) {
```

Comment:
I _think_ the way to "skip" a test if some condition is not met is to use `assumeTrue`.

`assumeTrue("Should not test snapshot version here", version != EsqlVersion.SNAPSHOT)`

Reply:
Please, ignore my comment. With this approach the test will always be skipped.